### PR TITLE
Metadata spreadsheet submission

### PR DESCRIPTION
**Pull request by jacoblurye**
...for submitting assay metadata spreadsheets to CIDC admins via email.

### CODE REVIEW

**Comment by jacoblurye** [comment]
Per @jim-bo's suggestion, I'm going to be updating this PR to upload the assay metadata to GCS and provide a GCS URI in the email, rather than attaching the metadata spreadsheet directly to the email.

File: cidc_api/shared/emails.py [hunk 1]
```diff
@@ -112,3 +112,32 @@ def new_upload_alert(upload, full_metadata) -> dict:
...
+    """
+    subject = (
+        f"[METADATA SUBMISSION]({ENV}) {user.email} submitted {trial_id}/{assay_type}"
+    )
+    html_content = f"""
+    <p><strong>user:</strong> {user.first_n} {user.last_n} ({user.email})</p>
+    <p><strong>contact email:</strong> {user.contact_email}</p>
+    <p><strong>protocol identifier:</strong> {trial_id}</p>
+    <p><strong>assay type:</strong> {assay_type}</p>
+    <p><strong>metadata spreadsheet GCS URI:</strong> {xlsx_gcs_uri}</p>
```
Comment (curlup) [inline cidc_api/shared/emails.py:133]: nitpick
```suggestion
    <p><strong>metadata spreadsheet GCS URI:</strong> <a href="{xlsx_gcs_uri}">{xlsx_gcs_uri}</a></p>
```
or something

**Review by curlup — APPROVED** [review]

**Comment by curlup** [comment]
@jacoblurye do you think it makes sense to alter the threshold on "Function X has 5 arguments" code climate checks maybe?

File: cidc_api/shared/emails.py (hunk 1 above)
Comment (jacoblurye) [reply cidc_api/shared/emails.py:133]: As currently implemented `xlsx_gcs_uri` will be `gs://...` url, not an HTTP url, but I like the general idea.

**Pull request merged**